test/sql/base.pg

```diff
@@ -454,6 +454,59 @@ SELECT has_function( 'table_version', 'ver_version'::name );
 
 SELECT has_function( 'table_version', 'ver_enable_versioning', ARRAY['regclass'] );
 
+-- Added after problem found in admin_bouldaries_uploader
```

Comment:
Maybe add a link here to the issue, to have a background of the error condition ?